package.json

```diff
@@ -19,7 +19,7 @@
     "@hookform/resolvers": "^3.3.1",
     "classnames": "^2.3.2",
     "react-hook-form": "^7.46.2",
-    "react-to-print": "^2.14.13",
+    "react-to-print": "^3.0.0-beta-1",
```

Comment:
Thank you @ibacher 

I didn't intend to do this migration since we might not be ready for it.
As you are aware, things are slightly different in beta v3, meaning i will have to refactor most of the printing logic in other parts of patient chart like vitals, medications, lab orders, etc.
I was testing if beta v3 actually solves the `Type Error: can not read properties of undefined` we experienced in #1847

Such migrations are the reasons why we might need a hook, wrapper or something like that to reduce the burden of writing alot of code when the react-to-print library hook parameters change
cc @brandones

Reply:
@denniskigen, beta v3 solves solves the Type Error: can not read properties of undefined on the first printing attempt

Reply:
Did you not notice that this version of the PR already does that refactoring?